examples/pytorch/question-answering/run_qa.py

```diff
@@ -571,9 +583,14 @@ def post_processing_function(examples, features, predictions, stage="eval"):
     def compute_metrics(p: EvalPrediction):
         return metric.compute(predictions=p.predictions, references=p.label_ids)
 
+    # Load possible existing recipe and new one passed in through command argument
+    existing_recipe = load_recipe(model_args.model_name_or_path)
```

Comment:
is there possible overlap between the recipes?

Reply:
possible, but it won't break with the current use cases (eval and train dense, pruned and quantized BERT). IMO, This should better be addressed at the manager level code.